build_scripts/travis_script.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""A simple script to run Travis build steps."""
+
+from __future__ import print_function
+
+import collections
+import os
+import sys
+
+import build_utils
+
+def _ismod(f):
```

Comment:
what is _ismod? probably needs a more descriptive name.

Reply:
[Just leaving a note for now as it might change based on how the other comments go.]

I wanted a short name so that it does force the line on which it is used to be split into multiple lines.